support/android/openssl.sh

```diff
@@ -221,7 +221,7 @@ xCFLAGS="-DSHARED_EXTENSION=.so -DOPENSSL_PIC -DDSO_DLFCN -DHAVE_DLFCN_H \
   -L$ANDROID_NDK_ROOT/platforms/$_ANDROID_API/$_ANDROID_ARCH/usr/lib -L$ANDROID_NDK_ROOT/toolchains/$_ANDROID_GCC-4.9/prebuilt/$host/lib/gcc/$_ANDROID_TARGET/4.9.x/ \
   --gcc-toolchain=$ANDROID_GCC_TOOLCHAIN \
   --sysroot=$ANDROID_SYSROOT \
-  --target=$RUST_TARGET
+  --target=$RUST_TARGET"
```

Comment:
roll this into 95056c98f69f01ae45270c1d4a39f30a33a81299 ?

Reply:
Will do when I squash the PR.